frontend/src/router.js

```diff
@@ -327,7 +335,11 @@ async function requireCamp(to, from, next) {
       next({ query: to.query })
     })
     .catch(() => {
-      next({ name: 'home' })
+      next({
+        name: 'PageNotFound',
+        params: [to.fullPath, ''],
+        replace: true,
+      })
```

Comment:
Since I redirect to an asterisk route the path is declared as params, if params[1] is not set there is a vue-warn in the console. if [1] is a string, it gets appended to the route path.